src/test/java/org/junitpioneer/jupiter/DefaultTimeZoneTests.java

```diff
@@ -88,7 +88,7 @@ void setsTimeZoneFromFullName() {
 	@Nested
 	@DefaultTimeZone("GMT-8:00")
 	@DisplayName("when applied on the class level")
-	class ClassLevelTestCase {
+	class ClassLevelTestCases {
```

Comment:
Renamed this class and `ExplicitGmtClassLevelTestCases` below even though they are directly executed by JUnit instead of through `PioneerTestKit`.